<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JumpToObservation</class>
 <widget class="QDialog" name="JumpToObservation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>232</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Jump to Observation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="labelStation">
     <property name="text">
      <string>Station:</string>
     </property>
     <property name="buddy">
      <cstring>textStation</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="textStation"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelParam">
     <property name="text">
      <string>Parameter:</string>
     </property>
     <property name="buddy">
      <cstring>comboParam</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboParam"/>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>73</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelType">
     <property name="text">
      <string>Type:</string>
     </property>
     <property name="buddy">
      <cstring>comboType</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="comboType"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelLevel">
     <property name="text">
      <string>Level:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="comboLevel"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelSensor">
     <property name="text">
      <string>Sensor:</string>
     </property>
     <property name="buddy">
      <cstring>spinSensorNr</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="spinSensorNr"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelObsTime">
     <property name="text">
      <string>Obs. Time:</string>
     </property>
     <property name="buddy">
      <cstring>editObsTime</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="MiDateTimeEdit" name="editObsTime">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::YearSection</enum>
     </property>
     <property name="displayFormat">
      <string notr="true">yyyy-MM-dd hh:mm</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonJump">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Jump</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>textStation</zorder>
  <zorder>labelParam</zorder>
  <zorder>comboParam</zorder>
  <zorder>labelType</zorder>
  <zorder>comboType</zorder>
  <zorder>labelObsTime</zorder>
  <zorder>editObsTime</zorder>
  <zorder>labelStation</zorder>
  <zorder>labelSensor</zorder>
  <zorder>spinSensorNr</zorder>
  <zorder>labelLevel</zorder>
  <zorder>comboLevel</zorder>
  <zorder>horizontalSpacer_2</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MiDateTimeEdit</class>
   <extends>QDateTimeEdit</extends>
   <header>util/MiDateTimeEdit.hh</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>JumpToObservation</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonJump</sender>
   <signal>clicked()</signal>
   <receiver>JumpToObservation</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onStationEdited()</slot>
  <slot>onParameterSelected(int)</slot>
 </slots>
</ui>
